package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Backend service js starter pack",
   "main": "./src/server.js",
   "scripts": {
-    "start": "NODE_ENV=production node ./src/server",
+    "start": "nodemon ./src/server",
```

Comment:
`nodemon` is a dev dependency and not a production one. As you can see below, the script for dev is `dev`. The prod start script should remain as-is.